src/test/groovy/com/gooddata/md/visualization/VisualizationObjectTest.groovy

```diff
@@ -41,67 +41,73 @@ class VisualizationObjectTest extends Specification {
+                                    null,
+                                    null
+                            ),
+                            new Measure(
+                                    measureDefinition,
+                                    'measure1',
+                                    'Measure 1 alias',
+                                    'Measure 1',
+                                    null
+                            )
+                    ])
+            ]
+        vizObject.filters = [
+                new PositiveAttributeFilter( new UriObjQualifier("/uri/to/displayForm/3"), ["ab", "cd"]),
+                new NegativeAttributeFilter( new UriObjQualifier("/uri/to/displayForm/3"), []),
+                new AbsoluteDateFilter( new UriObjQualifier("/uri/to/dataSet/1"), new LocalDate("2000-08-30"), new LocalDate("2017-08-07")),
+                new RelativeDateFilter( new UriObjQualifier("/uri/to/dataSet/2"), "month", null, null)
+            ]
+        vizObject.properties = '{"key":"value"}'
+        vizObject.referenceItems = references
+
         expect:
-        that new VisualizationObject(
-                new VisualizationObject.Content(
-                        new UriObjQualifier("visClass"),
-                        [
-                                new Bucket("bucket1", [attribute1]),
-                                new Bucket("bucket2", [
-                                        new Measure(
-                                                new VOSimpleMeasureDefinition(new UriObjQualifier("/uri/to/measure/0"), null, null, null),
-                                                "measure0",
-                                                "Measure 0 alias",
-                                                null,
-                                                null
-                                        ),
-                                        new Measure(
-                                                measureDefinition,
-                                                "measure1",
-                                                "Measure 1 alias",
-                                                "Measure 1",
-                                                null
-                                        )
-                                ])
-                        ],
-                        [
-                                positiveAttributeFilter,
-                                negativeAttributeFilter,
-                                absoluteDateFilter,
-                                relativeDateFilter
-                        ],
-                        "{\"key\":\"value\"}",
-                        references
-                ),
-                new Meta("complex")
-        ), jsonEquals(complexVisualization)
+        that vizObject, jsonEquals(resource(COMPLEX_VISUALIZATION))
     }
 
-    def "should serialize"() {
+    def "should serialize simple"() {
+        setup:
```

Comment:
we use `given` so pls don't mix `setup` in

Reply:
fixed